db/db_impl/db_impl_write.cc

```diff
@@ -2282,6 +2287,12 @@ size_t DBImpl::GetWalPreallocateBlockSize(uint64_t write_buffer_size) const {
       bsize = std::min<size_t>(bsize, buffer_size);
     }
   }
+  for (auto manager : cf_based_write_buffer_manager_) {
```

Comment:
It should be min(db_write_buffer_size, sum(cf_wbm_size) + db_wbm_size).

Reply:
Fixed